hatchet/tests/hpctoolkit_latest.py

```diff
@@ -0,0 +1,194 @@
+from hatchet import GraphFrame
+from hatchet.node import Node
+
+
+def test_import_entire_db(data_dir: str) -> None:
+    graphframe = GraphFrame.from_hpctoolkit_latest(f"{data_dir}/hpctoolkit-gamess")
+
+    assert len(graphframe.graph.roots) == 1
+    assert graphframe.graph.roots[0]._hatchet_nid == 1195
+    assert graphframe.graph.roots[0].frame["depth"] == 0
```

Comment:
`depth` is an attribute of a `Node` not a `Node.frame`. So it could be checked this way:
```suggestion
    assert graphframe.graph.roots[0]._depth == 0
```

Reply:
@michaelmckinsey1 fixed